src/js/themes/base.js

```diff
@@ -1489,6 +1489,23 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         margin: { right: 'xsmall' },
       },
       separator: ' : ',
+      size: {
+        xsmall: { 
+          pad: { horizontal: 'small' },
```

Comment:
Does this padding align with Button for its sizes?

Reply:
Tag has less horizontal padding (that's how it was spec'd in the figma). Button also calculates it as something like ${baseSpacing / 4 - borderWidth} etc rather than using tshirt sizes which we could do if needed, especially once the Figma details are decided.

Reply:
I tried to align them to be more consistent. They are a little different than button but that seems reasonable since they better fit the original Tag "medium" design.